ocs_ci/deployment/fusion_access.py

```
@@ -0,0 +1,300 @@
+    DEPLOYMENT section (optional overrides):
+        fusion_access_channel (str): Operator subscription channel.
+            Defaults to ``defaults.FUSION_ACCESS_CHANNEL``.
+        fusion_access_catalog_image (str): Override the catalog source image.
+        fusion_access_skip_cr (bool): When True, skip FusionAccessSAN CR creation
+            (useful when the CR is managed externally). Default False.
+
+    ENV_DATA section:
+        kubeconfig (via ``config.RUN["kubeconfig"]``): Path to the kubeconfig file.
+    """
+
+    def __init__(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)